actions/build.py

```diff
@@ -19,7 +19,7 @@ def run(self, packerfile, cwd=None, exclude=None, only=None, variables=None,
             p_result = p.build(parallel=parallel, debug=debug, force=force)
             result = self.format_results(p_result.stdout)
         except Exception as e:
-            result = (False, self.format_results(e.stdout, True))
+            result = (False, self.format_results(e.stdout, True))  # pylint: disable=no-member
```

Comment:
you probably just need to move the format_results() function above the run() function i'm guessing

Reply:
also, you probably just want to do `result = (False, self.format_results(e.stdout))`